tests/rptest/tests/usage_test.py

```diff
@@ -318,7 +318,9 @@ def delta_parse(x):
         fire_times = [delta_parse(x) for x in log_lines]
 
         # Ensure all deltas are 0 meaning there is no skew
-        return all([x == 0 for x in fire_times])
+        # Make an exception for 1 second delta to account for rounding skew
+        # when casting time now in epoch to seconds
+        return all([x == 0 or x == 1 for x in fire_times])
```

Comment:
>Also should we have a better error message? The linked bug just reports an "AssertionError" with no other information.

100%

Reply:
> Would it be better to just truncate instead of rounding to the nearest whole number?

After reading the docs i think actually that is whats going on? If that is the case i do not know how to explain a skew of 1 second then. 

https://en.cppreference.com/w/cpp/chrono/duration/duration_cast

BTW agreed on the error msg